app/models/course.rb

```diff
@@ -159,4 +159,21 @@ def funding
       'fee'
     end
   end
+
+  def publish_sites
+    site_statuses.status_new_status.each(&:status_running!)
```

Comment:
interestingly we had that to start with. I defer to @misaka

Reply:
If we need the `updated_at` timestamp to be updated, we can keep this as-is, otherwise I see no reason not to use `update_all`.

Since we don't have many site statuses per course, I don't think this will make a big difference though.

Reply:
there is an issue with `update_all` not calling the `TouchCourse` concern and not updating `updated_at`

Reply:
Ah ok, good reason to stick with something that fires callbacks then. 👍